### ISSUE TITLE
Class GeneaLabs\MixPanel\MixPanel does not exist

### ISSUE THREAD

**Issue by emergingdzns** [GeneaLabs/laravel-mixpanel#5]
I think the documentation may be old and using the previous version. I'm getting the following error when attempting to instantiate the mix panel library:
```
ReflectionException in Container.php line 790:
Class GeneaLabs\MixPanel\MixPanel does not exist
```
I tried changing it to the following which seemed more logical given the file names, but it still said not found:
```
Use GeneaLabs\LaravelMixpanel
```
Appreciate the pointers...

Thanks for your help!

**Comment by emergingdzns** [comment]
Ah. I found it.
```
Use GeneaLabs\LaravelMixpanel\LaravelMixpanel
```
And then in the __construct, change it like this:
```
public function __construct(LaravelMixpanel $mixPanel)
```

**Comment by mikebronner** [comment]
This will be fixed in version 0.4.6, in the next few minutes.
